surveys/templates/partials/sidebar.html

```diff
@@ -26,13 +26,13 @@ <h3><b>Just Spaces</b></h3>
                 </ul>
             </li>
             <li>
-                <a href="{% url 'surveys-list' %}">
+                <a href="{% url 'surveys-list' %}?published=f">
```

Comment:
I typically prefer to use `true` and `false` for boolean values instead of `t` and `f` (or `0` and `1`). There's no functional difference, it just makes the URL easier for me to read. Would be curious to hear your thoughts if you feel differently. Some discussion on SO: https://stackoverflow.com/q/21254159